plugin/output/s3/s3.go

```diff
@@ -74,13 +82,28 @@ type Config struct {
 	Endpoint  string `json:"endpoint" required:"true"`
 	AccessKey string `json:"access_key" required:"true"`
 	SecretKey string `json:"secret_key" required:"true"`
-	Bucket    string `json:"bucket" required:"true"`
-	Secure    bool   `json:"secure" default:"false"`
+	// Required s3 default bucket.
+	Bucket string `json:"bucket" required:"true"`
+	// MultiBuckets is additional buckets, which can also receive event.
+	// Event must contain `bucket_name` which value is valid s3 bucket name.
+	// Events without `bucket_name` sends to	 Bucket.
+	MultiBuckets []singleBucket `json:"multi_buckets" required:"false"`
+	Secure       bool           `json:"secure" default:"false"`
 
 	// for mock client injection
 	client *objectStoreClient
 }
 
+type singleBucket struct {
```

Comment:
Since this structure is related only to the config, i guess it is better to rename it to `singleBucketConfig`, what do you think?

Reply:
I'm ok with that, `singleBucket` hasn't any attached methods it's in fact DTO.